compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/resolve-react-hooks-based-on-import-name.expect.md

```diff
@@ -0,0 +1,79 @@
+
+## Input
+
+```javascript
+import { useState as useReactState } from "react";
+
+function Component() {
+  const [state, setState] = useReactState(0);
+
+  const onClick = () => {
+    setState((s) => s + 1);
+  };
+
+  return (
+    <>
+      Count {state}
+      <button onClick={onClick}>Increment</button>
+    </>
+  );
+}
+
+export const FIXTURE_ENTRYPOINT = {
+  fn: Component,
+  params: [{}],
+};
+
+```
+
+## Code
+
+```javascript
+import { c as _c } from "react/compiler-runtime";
+import { useState as useReactState } from "react";
+
+function Component() {
+  const $ = _c(4);
+  const [state, setState] = useReactState(0);
+  let t0;
+  if ($[0] === Symbol.for("react.memo_cache_sentinel")) {
+    t0 = () => {
+      setState((s) => s + 1);
```

Comment:
even though it's aliased on import, we know that `useReactState` is actually `useState`, so `setState` is non-reactive